File: src/main/java/me/yoursole/ctf/Events/RespawnEvent.java
package me.yoursole.ctf.Events;

import me.yoursole.ctf.DataFiles.GameData;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerRespawnEvent;

public class RespawnEvent implements Listener {
    @EventHandler
    public void onPlayerRespawn(PlayerRespawnEvent e){
        if(GameData.it==null)
            return;
        e.setRespawnLocation(GameData.gameSpawnPoint);
    }
}

File: src/main/kotlin/me/yoursole/ctf/events/RespawnEvent.kt
package me.yoursole.ctf.events

import me.yoursole.ctf.datafiles.GameData
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerRespawnEvent

object RespawnEvent : Listener {
    @EventHandler
    fun onPlayerRespawn(e: PlayerRespawnEvent) {
        if (GameData.it == null) return
        e.respawnLocation = GameData.gameSpawnPoint!!
    }
}